proc.c

```diff
@@ -439,13 +439,13 @@ scheduler(void)
         switch (pr->state)
         {
           case RUNNING:
-            pr->rutime++;
+            pr->rutime += (delta_ticks / QUANTA);
```

Comment:
A] i believe the handler just increments the ticks. forcing the proc to yield is done later on [on line 105]
B] i dunno. that's the checks that were there before, so i left them as is. if you compare it the default trap.c, you'll see that i really didn't change much